subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,76 @@
+
+#### Comprehension
+
+##### Describe the microservices architecture implemented.
```

Comment:
To be inline with the rest of the questions, I would change `Describe..` with `Ask the student to describe...`